File: epi_judge_python_solutions/euclidean_gcd.py
from test_framework import generic_test


def gcd(x: int, y: int) -> int:

    return x if y == 0 else gcd(y, x % y)


if __name__ == '__main__':
    exit(generic_test.generic_test_main('euclidean_gcd.py', 'gcd.tsv', gcd))

File: epi_judge_java/epi/EuclideanGcd.java
package epi;
import epi.test_framework.EpiTest;
import epi.test_framework.GenericTest;
public class EuclideanGcd {
  @EpiTest(testDataFile = "gcd.tsv")

  public static long GCD(long x, long y) {
    // TODO - you fill in here.
    return 0;
  }

  public static void main(String[] args) {
    System.exit(
        GenericTest
            .runFromAnnotations(args, "EuclideanGcd.java",
                                new Object() {}.getClass().getEnclosingClass())
            .ordinal());
  }
}
